packages/perseus/src/widgets/interactive-graphs/graphs/components/movable-point-view.tsx

```diff
@@ -0,0 +1,130 @@
+import {color as WBColor} from "@khanacademy/wonder-blocks-tokens";
```

Comment:
For posterity, could you add a small docblock at the top of this file explaning what MovablePointView is and where/how it's used?